neqo-transport/src/connection/tests/idle.rs

```diff
@@ -53,6 +53,13 @@ fn test_idle_timeout(client: &mut Connection, server: &mut Connection, timeout:
     assert!(matches!(client.state(), State::Closed(_)));
 }
 
+#[test]
+fn init_rtt_configuration() {
+    const CUSTOM_INIT_RTT: Duration = Duration::from_millis(200);
+    let client = new_client(ConnectionParameters::default().initial_rtt(CUSTOM_INIT_RTT));
+    assert_eq!(client.conn_params.get_initial_rtt(), CUSTOM_INIT_RTT);
```

Comment:
Is there a way to test whether the different initial RTT actually causes the desired behavior to happen?

Reply:
Hmm, for that I guess I have to get more familiar with the main consumer of the configuration, `Connection`?

I had a look at how this parameter is used inside the `Connection` structure and I found some usages where I made adaptions. The changes are inside https://github.com/mozilla/neqo/pull/2041/commits . The next step would be to find a way to verify that a different value actually leads to different behavior..